tests/test_utils/test_moving.py

```
@@ -0,0 +1,261 @@
+        # Test properties only where values are finite
+        mean_vals = results["mean"][finite_indices]
+        sum_vals = results["sum"][finite_indices]
+        min_vals = results["min"][finite_indices]
+        max_vals = results["max"][finite_indices]
+
+        # Sum should be >= mean (for positive window size)
+        assert np.all(sum_vals >= mean_vals)
+        # Min should be <= max
+        assert np.all(min_vals <= max_vals)
+
+    def _compare_engine_results(self, result1, result2, window, operation):
```

Ruff (0.13.1)

Unused method argument: `operation`

(ARG002)